### DISCUSSION TITLE
Maybe try adding to the documentation /claim is a command because it has every command except for the very basic one. not very complete documentation :(

### DISCUSSION THREAD

**Discussion by trevluv** [GriefPrevention/GriefPrevention#1287] [Ideas]
Maybe try adding to the documentation /claim is a command because it has every command except for the very basic one. not very complete documentation :(

**Comment by trevluv** [comment]
lmao I have been trying to make a fucking claim for 30 minutes lmao /claim only works on checks lmao wtf is wrong with your documentation .

You LITTERALY CANNOT FIND /claim in your documentation LOLOLOLOL

**Comment by bigpresh** [comment]
The majority of users create claims by right-clicking with their golden shovel to place the claim exactly where they want it, I don't think many people actually use `/claim`.  Some servers will set it up such that you can *only* do it with the golden shovel, so that you have to have expended some effort to mine and find gold before you can make your first claim (other than the automatic claim around your first chest).

That said - yes, `/claim` very much should be documented.  I'll sort that, thanks.

**Comment by RoboMWM** [comment]
bigscary hasn't documented new things that he added the last couple years before he "retired" from MC.

Thanks for the condescending remarks tho, I'll document it when I feel like it, or you can paste an example here for me to paste in.
